api/lib/domain/usecases/get-pole-emploi-sendings.js

```diff
@@ -1,3 +1,17 @@
-module.exports = function getPoleEmploiSendings({ poleEmploiSendingRepository }) {
-  return poleEmploiSendingRepository.get();
+const poleEmploiService = require('../services/pole-emploi-service');
+
+module.exports = async function getPoleEmploiSendings({ cursor, poleEmploiSendingRepository }) {
+  const cursorData = await poleEmploiService.decodeCursor(cursor);
+  const sendings = await poleEmploiSendingRepository.get(cursorData);
+  const link = _generateLink(sendings);
+  return { sendings, link };
 };
+
+function _generateLink(sendings) {
+  if (sendings.length > 0) {
+    const lastSending = sendings[sendings.length - 1];
+    const link = poleEmploiService.generateLink(lastSending);
+    return link;
+  }
+  return null;
+}
```

Comment:
**suggestion:** Je propose ici le ["return early patterm"](https://dev.to/jpswade/return-early-12o5)

```suggestion
function _generateLink(sendings) {
  if (!sendings.length) return null;
  
  const lastSending = sendings[sendings.length - 1];
  const link = poleEmploiService.generateLink(lastSending);
  return link;
}
```